packages/components/react/src/core.ts

```diff
@@ -0,0 +1,3 @@
+export * from './core/CodeMirrorEditor/index.js';
+export * from './core/FileTree.js';
+export * from './core/Terminal/index.js';
```

Comment:
As this now becomes part of the library's public API, I would recommend to be explicit about the exports here. It's better to be verbose with these exports so that we'll know which parts were "promised" to be present.

```suggestion
export { CodeMirrorEditor } from './core/CodeMirrorEditor/index.js'; // and maybe the types too if they were intended?
export { FileTree } from './core/FileTree.js';
export { Terminal, type TerminalRef } from './core/Terminal/index.js';

```

Reply:
Oh well, `packages/components/react/src/index.ts` is also exporting quite much using wildcards 🤔 
Feel free to leave this as is, but at some point I think we'll need to check which entrypoints are exposed to users. Right know this is quite prone to accidental breaking changes.

Reply:
That's a very good point! I'll add explicit imports for this file, we can revisit the other imports in separate PRs.